### PR TITLE
python api for downloading embeddings and pred_probs

### PR DESCRIPTION
**Pull request by clu0**
Expose api for downloading pred_probs and embeddings from training for cleansets

corresponding PRs:
training (save pred_probs and embeddings): https://github.com/cleanlab/cleanlab-studio-training/pull/211
scheduler (send user_id to training): https://github.com/cleanlab/cleanlab-studio-scheduler/pull/64
backend (api to download): https://github.com/cleanlab/cleanlab-studio-backend/pull/535

### CODE REVIEW

**Review by axl1313 — COMMENTED** [review]
Is the corresponding backend for this implemented yet?

**Comment by clu0** [comment]
> Is the corresponding backend for this implemented yet?

ah yes sorry forgot to link to them, I added them to the description and tagged you in training and backend

**Review by axl1313 — APPROVED** [review]
lgtm! just one non-blocking question

File: cleanlab_studio/cli/types.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ class DatasetFileExtension(Enum):
     xls = ".xls"
     xlsx = ".xlsx"
     json = ".json"
+    pkl = ".pkl"
```
Comment (axl1313) [inline cleanlab_studio/cli/types.py:23]: is this used?

Comment (clu0) [reply cleanlab_studio/cli/types.py:23]: ah not anymore. this class is actually used in the backend for one of the download functions, but i'm no longer using that

thanks, i'll remove

**Pull request merged**